app/controllers/facility_statements_controller.rb

```diff
@@ -18,16 +18,13 @@ def initialize
 
   # GET /facilities/:facility_id/statements
   def index
-    @statements =
-      current_facility
-      .statements
-      .order(created_at: :desc)
-      .paginate(page: params[:page])
+    statements = current_facility.cross_billing? ? Statement.all : current_facility.statements
+    @statements = statements.order(created_at: :desc).paginate(page: params[:page])
   end
 
   # GET /facilities/:facility_id/statements/new
   def new_with_search
-    @order_details = @order_details.need_statement(@facility)
+    @order_details = current_facility.cross_billing? ? @order_details.need_any_statement : @order_details.need_statement(@facility)
```

Comment:
This might tie into the null object idea, but I think this might be better if we do the cross-facility checking into `need_statement` rather than need to do the check several places.